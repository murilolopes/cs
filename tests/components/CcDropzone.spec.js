import { mount, shallowMount } from '@vue/test-utils'
import CCDropzone from '@core/components/cc-dropzone/CcDropzone.vue'

describe('CCDropzone.vue', () => {
  it("should render correctly", () => {
    const wrapper = shallowMount(CCDropzone);
    expect(wrapper.element).toMatchSnapshot();
  });

  it('Should', () => {
    const { vm } = mount(CCDropzone)

    expect(vm.isDragging).toEqual(false)
    expect(vm.files).toEqual([])
    expect(vm.hasFormatError).toEqual(false)
  })
})
