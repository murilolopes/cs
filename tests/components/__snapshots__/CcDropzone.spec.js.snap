// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CCDropzone.vue should render correctly 1`] = `
<div
  class="cc-dropzone cc-dropzone-min-height"
>
  <div
    class="dropzone-container"
  >
    <input
      accept=".xml"
      class="hidden-input"
      id="fileInput"
      name="file"
      required="required"
      type="file"
    />
     
    <label
      class="file-label"
      for="fileInput"
    >
      <h4
        class="text-primary"
      >
        Arraste seu arquivo at√© aqui ou clique para procurar.
      </h4>
       
      <span>
        Envie seu arquivo no formato XML.
      </span>
    </label>
     
    <!---->
  </div>
</div>
`;
